upgrade.md

```diff
@@ -9,10 +9,31 @@
 
 <a name="upgrade-5.0.16"></a>
 ## Upgrading To 5.0.16
+ 
 
-In your `bootstrap/autoload.php` file, update the `$compiledPath` variable to:
+In Your `bootstrap/autoload.php` file, change the `$compiledPath` variable :
 
-	$compiledPath = __DIR__.'/../vendor/compiled.php';
+ From (old path):
```

Comment:
Hay Graham Campbell :) 

Please node that compiled.php MUST be removed !! 
Otherwise you will not able  to update your Project. 

That All :)